### PR TITLE
added user_local.props and added DynamoServices

### PR DESCRIPTION
**Pull request by rk-lakshmanan**
Purpose
Reference: [MAGN-9811](http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-9811)
This PR is to include DynamoServices reference to the project.

Reviewer
@sharadkjaiswal

### CODE REVIEW

**Comment by ikeough** [comment]
@rk-lakshmanan No. We should be using the Dynamo NuGet packages for this. See my comments on the internal task.

File: DynamoText.csproj [hunk 1]
```diff
@@ -1,6 +1,9 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+   <ImportGroup Label="PropertySheets">
+    <Import Project="$(SolutionDir)/Config/user_local.props" />
```
Comment (sharadkjaiswal) [inline DynamoText.csproj]: We should include CS.props, which should be using nuget package path.

**Comment by ikeough** [comment]
@sharadkjaiswal Not sure I understand the comment above. With NuGet you shouldn't have to import anything into the project file. It'll insert whatever imports it needs. What's the aim here?

**Comment by sharadkjaiswal** [comment]
@ikeough, having a dependency on nuget file will not tell us about when we break the build due to change in APIs. So internal developers as well as CI must build with Dynamo\bin binaries. So when Dynamo\bin is available it will reference the binaries from there otherwise it will reference binaries from nuget package. Please see https://git.autodesk.com/keoughi/Dynamo-Solar-Analysis/pull/7/

**Comment by ikeough** [comment]
@sharadkjaiswal I commented on the other PR. How will we update the nuget package versions referenced by these projects? If we use the "manage nuget packages" in VS,  those will be written into separate references in the .csproj....I think. Then we'll have two references, the one created by nuget, and the one loaded by the .props. Which one wins?

**Pull request closed without merging**